src/scene_battle_rpg2k3.cpp

```diff
@@ -261,29 +261,39 @@ void Scene_Battle_Rpg2k3::Update() {
 
 	switch (state) {
 		case State_SelectActor:
-		case State_AutoBattle: {
+		case State_AutoBattle:
+		case State_SelectCommand:
+		case State_SelectEnemyTarget:
+		case State_SelectAllyTarget: {
+			if (Game_System::GetAtbMode() == lcf::rpg::SaveSystem::AtbMode_atb_wait && (state == State_SelectCommand || state == State_SelectEnemyTarget || state == State_SelectAllyTarget)) break;
+			if ((state == State_SelectEnemyTarget || state == State_SelectAllyTarget) && (previous_state == State_SelectSkill || previous_state == State_SelectItem)) break;
```

Comment:
You check "case" values in an if and then break out of the switch.
I suggest putting them before ``State selectactor`` and then just ``// fallthrough``

Reply:
I'm probably too stupid but how I'm supposed to do the fallthrough?

Reply:
To fallthrough just don't use break. It is just convention (compilers warn about it) that you must use ``// fallthrough`` (a comment) when the fallthrough was intended.

Something like this.

```
case State_SelectEnemyTarget:
case State_SelectAllyTarget:
  if (previous_state == State_SelectSkill || previous_state == State_SelectItem) {
    break;
  }
  // fallthrough
case State_SelectCommand:
  if (Game_System::GetAtbMode() == lcf::rpg::SaveSystem::AtbMode_atb_wait) {
    break;
  }
  // fallthrough
case State_SelectActor:
case State_AutoBattle:
 /* more code */
```

Reply:
Thank you for your input! It helps me understanding the switch-case-operation a bit better. I have updated the PR.